pkgs/development/python-modules/pyro4/default.nix

```diff
@@ -1,20 +1,22 @@
 { lib
 , buildPythonPackage
-, fetchPypi
-, serpent
-, dill
 , cloudpickle
+, dill
+, fetchPypi
 , msgpack
-, isPy27
 , pytestCheckHook
+, pythonAtLeast
+, serpent
 }:
 
 buildPythonPackage rec {
   pname = "pyro4";
   version = "4.82";
   format = "setuptools";
 
-  disabled = isPy27;
+  # No support Python >= 3.11
+  # https://github.com/irmen/Pyro4/issues/246
+  disabled = pythonAtLeast "3.11";
```

Comment:
Shouldn't we disable only after the first issue with Python 3.11 has been found.

Reply:
Tests are failing with Python 3.11

```bash
error: builder for '/nix/store/1s257c409ja63b4blalpg9zrcmh1c6w8-python3.11-pyro4-4.82.drv' failed with exit code 1;
       last 10 log lines:
       > FAILED tests/PyroTests/test_serialize.py::SerializeTests_json::testCustomClassFail - AssertionError: error expected, shouldn't deserialize unknown class
       > FAILED tests/PyroTests/test_serialize.py::SerializeTests_json::testCustomClassOk - AssertionError: must fail
       > FAILED tests/PyroTests/test_serialize.py::SerializeTests_marshal::testCustomClassFail - AssertionError: error expected, shouldn't deserialize unknown class
       > FAILED tests/PyroTests/test_serialize.py::SerializeTests_marshal::testCustomClassOk - AssertionError: must fail
       > FAILED tests/PyroTests/test_serialize.py::SerializeTests_msgpack::testCustomClassFail - AssertionError: error expected, shouldn't deserialize unknown class
       > FAILED tests/PyroTests/test_serialize.py::SerializeTests_msgpack::testCustomClassOk - AssertionError: must fail
       > FAILED tests/PyroTests/test_serialize.py::GenericTests::testCustomDictClass - KeyError: '__class__'
       > FAILED tests/PyroTests/test_serialize.py::GenericTests::testDictClassFail - KeyError: '__class__'
       > = 8 failed, 593 passed, 20 skipped, 8 deselected, 860 warnings in 81.15s (0:01:21) =
```